### PR TITLE
Bar chart data

### PR DESCRIPTION
**Pull request by JiaoJiao123**
Related to Issue Bar graphs not shown correctly #1477

### CODE REVIEW

**Comment by zhx828** [comment]
@JiaoJiao123 could you do a rebase? And maybe squash to one commit as well?

File: portal/src/main/webapp/js/src/cbio-util.js [hunk 1]
```diff
@@ -552,9 +552,26 @@ cbio.util = (function() {
             maxValue = Math.ceil(q3 + iqr * 1.5);
             minValue = Math.floor(q1 - iqr * 1.5);
         }
-        if(maxValue > values[values.length - 1])maxValue = values[values.length - 1];
         if(minValue < values[0])minValue = values[0];
-        // Then return
+        if(maxValue > values[values.length - 1])maxValue = values[values.length - 1];
+        //provide the option to choose min and max values from the input array
```
Comment (zhx828) [inline portal/src/main/webapp/js/src/cbio-util.js]: Maybe include this explanation above the function. 
I would recommend jsdoc style
/**
 * ....
 * @param {Array} data 
 * @param {Boolean} inArrayFlag
 * @return {Array} outliers
 */

**Comment by zhx828** [comment]
Except the comments above, the code looks good to me.

**Comment by JiaoJiao123** [comment]
@jjgao The fix about clinical tab error in study view is added in this PR.

**Comment by jjgao** [comment]
@JiaoJiao123 @zhx828 is this ready to be merged?

**Comment by zhx828** [comment]
@jjgao Jiaojiao's fix should solve the clinical tab issue. I have another EFDT update coming up, but that that can be merged later.

**Comment by pieterlukasse** [comment]
@oplantalech : please also test on your side whether this solves the original problem you logged.

**Comment by oplantalech** [comment]
@pieterlukasse I've checked it and now it works correctly.

**Pull request merged**